posts/2022-01-04-azure-eventing.md

```diff
@@ -0,0 +1,496 @@
+
+<figure>
+  <img src="https://user-images.githubusercontent.com/85032783/147981825-f0e94c34-6488-4fd3-82f4-7ef8e00224a2.png" width="100%"/>
+  <figcaption align="center"><b>Figure 1</b><br/><i>A Supplier, Distributor, and Retailer reacting in real-time across clouds as changes to purchase orders occur</i></figcaption>
+</figure>
+
+We'll do exactly that after setting up the proper Vendia Share and Azure configuration.  The process will start by modifying the fulfillment date of a purchase order by executing a [GraphQL mutation](https://graphql.org/learn/queries/#mutations) against the Supplier's node in the Uni.  That, in turn, will cause an event to flow to Retailer's node in the Uni and then to the Retailer's Azure environment, delivered to [Azure Event Grid](https://azure.microsoft.com/en-us/services/event-grid/).  From there, an event-driven [Azure Function](https://azure.microsoft.com/en-us/services/functions/) within the Retailer's Azure environment will process the delivered event.
+
+Now let's get started!
+
+## Step 0 - Prerequisites
+
+To replicate the Retailer's environment on Azure, we'll first need to:
+
+* [Create a Vendia Share account](https://share.vendia.net/)
+* [Create an Azure account](https://azure.microsoft.com/en-us/free/)
+* Install the [Azure Command Line Interface](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) (CLI)
+* Install the [Vendia Command Line Interface](https://www.vendia.net/docs/share/cli) (CLI)
+
+## Step 1 - Create a Cross-Cloud Vendia Uni
+
+The first step is to create a cross-cloud Uni for this example, with the following participants:
+
+* A `Supplier` AWS Node (`us-east-2`)
+* A `Distributor` AWS Node (`us-west-2`)
+* A `Retailer` Azure Node (`eastus`)
+
+This is accomplished using a `registration.json` file that defines one Node per Uni participant.  Each Node definition includes a `csp` (short for Cloud Service Provider) and a `region`, which is [unique to each CSP](https://www.vendia.net/docs/share/cli/guide#supported-cloud-platforms-and-regions).  Using the example below, we're able to provision a multi-cloud, multi-region Uni that connects all three participants in just about 5 minutes.
+
+> Note: Please refer to our docs for more information about the modeling Uni participants using [a registration file](https://www.vendia.net/docs/share/uni-creation#registration-file) or modeling data using [a schema file](https://www.vendia.net/docs/share/data-modeling#sample-registration-files).
+
```

Comment:
Do we want to have the user create a directory for this example since there will several files created and we are asking that they put them all in the same directory?

Reply:
Omitted for brevity.